config/esw.yaml

```diff
@@ -46,11 +48,15 @@ science:
     baud_rate: 38400
     timeout: 0
     port: "/dev/ttyTHS0"
+  syringe_servo_positions:
+    site_A: 135
```

Comment:
can u at least add a comment that these are random numbers for now and that it should be changed